frontend/src/store/slices/accessRequests/accessRequestsReducer.test.tsx

```diff
@@ -0,0 +1,128 @@
+import { mockAccessRequest } from 'mocks/filterDataMock';
+import { accessRequestsSlice, initialState } from 'store/slices/accessRequests/accessRequestsSlice';
+
+const accessRequestReducer = accessRequestsSlice.reducer;
+describe('access request reducer functionality', () => {
```

Comment:
Created these reducer tests to validate that no functionality was changed between the old and new reducer by:
1) create these tests, execute against the old reducer.
2) create the new reducer.
3) swap these tests to point to the new reducer.
4) delete the old reducer.